src/Core/Components/Overflow/FluentOverflow.razor.cs

```diff
@@ -88,8 +90,16 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
         {
             _jsModule = await JSRuntime.InvokeAsync<IJSObjectReference>("import", JAVASCRIPT_FILE);
             _dotNetHelper = DotNetObjectReference.Create(this);
+            await _jsModule.InvokeVoidAsync("fluentOverflowInitialize", _dotNetHelper, Id, IsHorizontal, Selectors);
```

Comment:
Could you call `RefreshAsync` to have only one call to this JS method?

Reply:
No, the 'initialize code' is different JS call then for the 'refresh code'. Initialize sets up the MutationObserver and ResizeObserver and then calls the (re)calculation on the JS side. Resize just calls the (re)calculation

Reply:
Hups... I didn't have my glasses ;-)